apps/cowswap-frontend/src/modules/tradeQuote/utils/quoteUsingSameParameters.ts

```diff
@@ -14,28 +14,35 @@ import { AppDataInfo } from 'modules/appData'
 export function quoteUsingSameParameters(
   chainId: SupportedChainId,
   currentParams: Nullish<TradeParameters>,
-  nextParams: Nullish<TradeParameters>,
+  nextParams: Nullish<QuoteBridgeRequest>,
   currentAppData: AppDataInfo['doc'] | undefined,
   appData: AppDataInfo['doc'] | undefined,
 ): boolean {
   if (!currentParams || !nextParams) return false
 
+  /**
+   * TODO: in bridging mode buy token changes to intermediate token
+   * TODO: in bridging mode, appData contains only hooks without other information
+   * TODO: in bridging mode, receiver became a CoW Shed address
+   */
+  const isBridge = true
   const hasSameAppData = compareAppDataWithoutQuoteData(currentAppData, appData)
-  const isNativeToken = getIsNativeToken(chainId, nextParams.sellToken)
+  const isNativeToken = getIsNativeToken(chainId, nextParams.sellTokenAddress)
   const wrappedToken = WRAPPED_NATIVE_CURRENCIES[chainId]
   /**
    * Due to CoW Protocol design, we do
    */
-  const nextSellToken = isNativeToken ? wrappedToken.address.toLowerCase() : nextParams.sellToken.toLowerCase()
+  const nextSellToken = isNativeToken ? wrappedToken.address.toLowerCase() : nextParams.sellTokenAddress.toLowerCase()
 
+  // TODO: compare tokens chainIds
   return (
-    hasSameAppData &&
+    (isBridge ? true : hasSameAppData) &&
```

Comment:
Nitpick:

```suggestion
    (isBridge || hasSameAppData) &&
```

Same below

Reply:
Completely changed that part